libcob/common.c

```diff
@@ -2025,6 +2021,20 @@ cob_cmp_alnum (cob_field *f1, cob_field *f2)
 	return 0;
 }
 
+/* compare content of field 'f1' to content of 'f2', space padded,
+   using the optional collating sequence of the program */
+static int
+cob_cmp_alnum (cob_field *f1, cob_field *f2)
+{
+	const unsigned char	*col = COB_MODULE_PTR->collating_sequence;
+	const unsigned char	*data1 = COB_FIELD_DATA (f1);
+	const unsigned char	*data2 = COB_FIELD_DATA (f2);
+	const size_t		size1 = COB_FIELD_SIZE (f1);
+	const size_t		size2 = COB_FIELD_SIZE (f2);	
+	return cob_cmp_strings ((unsigned char *)data1, (unsigned char *)data2,
+		size1, size2, col);
```

Comment:
Is there a reason to not pass the arguments directly without intermediate variables?

Reply:
@engboris you've marked that as solved but the question is unanswered?